crates/compiler/test_gen/src/gen_num.rs

```diff
@@ -3998,3 +3998,49 @@ fn num_count_one_bits() {
     assert_evals_to!(r#"Num.countOneBits 0u32"#, 0, usize);
     assert_evals_to!(r#"Num.countOneBits 0b0010_1111u64"#, 5, usize);
 }
+
+#[test]
+#[cfg(any(feature = "gen-llvm", feature = "gen-dev", feature = "gen-wasm"))]
+fn num_abs_diff() {
+    assert_evals_to!(r#"Num.absDiff 0 0"#, 0, i64);
+    assert_evals_to!(r#"Num.absDiff 1 2"#, 1, i64);
+    assert_evals_to!(r#"Num.absDiff 2 1"#, 1, i64);
+    assert_evals_to!(r#"Num.absDiff -1 1"#, 2, i64);
+    assert_evals_to!(r#"Num.absDiff 1 -1"#, 2, i64);
+    assert_evals_to!(r#"Num.absDiff Num.minI64 -1"#, i64::MAX, i64);
+
+    assert_evals_to!(r#"Num.absDiff 0.0 0.0"#, 0.0, f64);
+    assert_evals_to!(r#"Num.absDiff 1.0 2.0"#, 1.0, f64);
+    assert_evals_to!(r#"Num.absDiff 2.0 1.0"#, 1.0, f64);
+    assert_evals_to!(r#"Num.absDiff -1.0 1.0"#, 2.0, f64);
+    assert_evals_to!(r#"Num.absDiff 1.0 -1.0"#, 2.0, f64);
```

Comment:
could you break this up into 2 functions (one for integers, one for floats)? rust runs these asserts sequentially, so this test will take a long time to complete. by breaking it up into smaller pieces, your cores are used more effectively

Reply:
Done in https://github.com/roc-lang/roc/pull/5275/commits/54992fb4367777162f171f511ef8d9a3798648cc